packages/explat-client/src/internal/experiment-assignments.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { ExperimentAssignment } from '../types';
+import * as Timing from './timing';
+
+/**
+ * Check if an ExperimentAssignment is still alive (as in the TTL).
+ *
+ * @param experimentAssignment The experiment assignment to check
+ */
+export function isAlive( experimentAssignment: ExperimentAssignment ): boolean {
+	return (
+		Timing.monotonicNow() <
+		experimentAssignment.ttl * Timing.MILLISECONDS_PER_SECOND +
+			experimentAssignment.retrievedTimestamp
+	);
+}
+
+/**
+ * The null ExperimentAssignment we return when we can't retrieve one.
+ */
+export const createNullExperimentAssignment = (): ExperimentAssignment => ( {
```

Comment:
Linking to the previous comment since this got moved: https://github.com/Automattic/wp-calypso/pull/48475#discussion_r565847939

Let's add the `experimentName` as a parameter of this function.